autoPlay.user.js

```diff
@@ -1230,6 +1249,22 @@ function useMaxElementalDmgIfRelevant() {
 	}
 }
 
+function useWormholeIfRelevant() {
+	// Check the time before using wormhole.
+	if (cHours != 15) {
+		return;
+	}
+	if (timeLeft > minsLeft) {
+		return;
+	}
+	// Check if Wormhole is purchased
+	if (canUseItem(ITEMS.WORMHOLE) && getActiveAbilityLaneCount(ITEMS.WORMHOLE) <= 0) {
+		// Wormhole is purchased, cooled down, and needed. Trigger it.
+		advLog('Less than 30 minutes for game to end. Triggering wormholes...', 2);
```

Comment:
We can concatenate here to give the proper minsLeft value.

advLog('Less than ' + minsLeft + ' minutes for game to end. Triggering wormholes...', 2);